app/flows/check_travel_during_coronavirus_flow/outcomes/_return_from_ireland_and_other_countries.erb

```diff
@@ -0,0 +1,25 @@
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "If you’ve been in Ireland for at 10 days or more before travelling to England",
+    heading_level: 3,
+    font_size: "m",
+    margin_bottom: 4,
+  } %>
+  <p class="govuk-body">There are no entry requirements if you’re travelling to England from Ireland after going to another country and have been in Ireland or anywhere else in the <a href="/government/publications/common-travel-area-guidance" class="govuk-link">Common Travel Area</a> for 10 days or more.</p>
+
+  <p class="govuk-body">You do not need to:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>take any COVID-19 tests before you travel to England or after you arrive</li>
+    <li>quarantine when you arrive</li>
+    <li>complete a passenger locator form</li>
+    <li>show proof of vaccination</li>
+  </ul>
+
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "If you’ve been in Ireland for less than 10 days before travelling to England",
+    heading_level: 3,
+    font_size: "m",
+    margin_bottom: 4,
+  } %>
+
+  <p class="govuk-body">If you’re travelling to England from Ireland and you’ve been in Ireland or anywhere else in the <a href="/government/publications/common-travel-area-guidance" class="govuk-link">Common Travel Area</a> for less than 10 days, you’ll need to follow the entry requirements below.</p>
+  <p class="govuk-body">If you’re travelling to England from another country and have been in Ireland before the other country, you need to follow the entry requirements below.</p>
```

Comment:
Clearly not a dev issue, but I find the duplicate use of `you need to follow the entry requirements below` a bit odd. Wondering why it can't be a single sentence 🤔

Reply:
😬 Content decision. Nothing to see here!